test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1902,18 +1902,18 @@ def test_overlap_resv(self):
         a = {ATTR_queue: r1name[0], 'Resource_List.select': '1:ncpus=1'}
         j1 = Job(TEST_USER, a)
         jid1 = self.server.submit(j1)
-        self.server.expect(JOB, {'job_state': 'Q'}, id=jid1)
+        self.server.expect(JOB, 'comment', op=SET, id=jid1)
```

Comment:
So I usually do both here.  I expect comment as set and then check the job in the Q state.  If you just check for the comment being set, the job could be in other states.  For example, if the job runs, the comment will be set.